File: RiveAnimationExt/src/main/java/com/softick/riveanimationext/MyRiveAnimationView.java
package com.softick.riveanimationext;

import android.content.Context;
import android.util.AttributeSet;
import android.view.MotionEvent;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import app.rive.runtime.kotlin.RiveAnimationView;

public class MyRiveAnimationView extends RiveAnimationView {
    public MyRiveAnimationView(@NonNull Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    @Override
    public boolean onTouchEvent(@Nullable MotionEvent event) {
        return false;
    }
}

File: RiveAnimationExt/src/main/java/com/softick/riveanimationext/MyRiveAnimationView.kt
package com.softick.riveanimationext

import android.content.Context
import android.util.AttributeSet
import android.view.MotionEvent
import app.rive.runtime.kotlin.RiveAnimationView

open class MyRiveAnimationView(context: Context, attrs: AttributeSet?) : RiveAnimationView(context, attrs)
{
    override fun onTouchEvent(event: MotionEvent?): Boolean {
        return false
    }
}